license-scanner/cli/scan.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from "license-scanner/constants";
 import { ensureDatabase, getSaveScanResultItem } from "license-scanner/db";
 import { getLicenseMatcher, loadLicensesNormalized, throwLicensingErrors } from "license-scanner/license";
-import { Logger, LogLevel } from "license-scanner/logger";
```

Comment:
The diff is hard to read but in summary what happens is:
- Reading arguments is removed
- More complex argument parsing is extracted into functions